source/_integrations/knx.markdown

```diff
@@ -1768,6 +1771,63 @@ entity_category:
   default: None
 {% endconfiguration %}
 
+## Time
+
+The KNX time platform allows to send time values to the KNX bus and update its state from received telegrams. It can optionally respond to read requests from the KNX bus with its current state.
+
+<div class='note'>
+
+Time entities without a `state_address` will restore their last known state after Home Assistant was restarted.
+
+Times having a `state_address` configured request their current state from the KNX bus.
+
+</div>
+
+<div class='note'>
+
+The `day` filed of the time telegram will always be set to 0 (`no day`).
+
+</div>
+
+```yaml
+# Example configuration.yaml entry
+knx:
+  time:
+    - name: "Time"
+      address: "0/0/2"
+      state_address: "0/0/2"
+```
+
+{% configuration %}
+name:
+  description: A name for this device used within Home Assistant.
+  required: false
+  type: string
+address:
+  description: Group address new values will be sent to. *DPT 10.001*
```

Comment:
```suggestion
  description: Group address. New values will be sent to *DPT 10.001*.
```

Reply:
Its the configured Group address we send to.
`DPT` is a common KNX term for "Data point type" and describes an attribute/requirement of that Group Address.

Reply:
Ah, I completely misunderstood that sentence then. Please disregard my change suggestions.